pages/_type/_id/version.vue

```diff
@@ -398,6 +398,7 @@
             <input
               v-model="newDependencyId"
               type="text"
+              oninput="this.value=this.value.replace(/[^a-z]/gi,'')"
```

Comment:
Should only remove whitespace

Reply:
Pushed a commit to this pr. Should be good now.